extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -116,7 +117,11 @@ export class AmpSidebar extends AMP.BaseElement {
     this.action_ = Services.actionServiceForDoc(element);
 
     if (this.side_ != 'left' && this.side_ != 'right') {
-      this.side_ = isRTL(this.document_) ? 'right' : 'left';
+      if (descendsFromStory(element)) {
```

Comment:
nit: To consolidate logic and display intent (and easier to test!) you can maybe structure it like this:

```js
  this.side_ = this.maybeFlip_(isRTL(this.document_) ? 'right' : 'left');
  // ...
  maybeFlip_(side) {
    if (!descendsFromStory(this.element) {
      return side;
    }
    return side == 'left' ? 'right' : 'left';
  }
```

also would be nice if we add constants for `LEFT` and `RIGHT` to let the compiler minify better.